.gitignore

```diff
@@ -19,6 +19,8 @@ docker-compose.override.yaml
 
```

Comment:
```
# .editorconfig
.editorconfig
```

Reply:
Updated and reverted dockerignore script